en/flight_controller/pixhack_v3.md

```diff
@@ -15,9 +18,9 @@ The board is a variant of the SOLO Pixhawk<sup>&reg;</sup> 2 (PH2) flight contro
   * STM32F427
   * STM32F100 (Failsafe co-processor)
 * Sensors:
-  * Accelerometers (3): LS303D, MPU6000, MPU9250/hmc5983
-  * Gyroscopes (3): L3GD20, MPU6000, MPU9250
-  * Compass (2): LS303D, MPU9250
+  * Accelerometers (2): LS303D, MPU6000
```

Comment:
Isn't it an mpu9250?

Reply:
@cuhome - Please list the devices on the pixhack v3 and v3x so it may be documented correctly.

Reply:
@cuhome I guess you're away on Chinese new year. This will wait for your return.